src/biz/bokhorst/xprivacy/PrivacyService.java

```diff
@@ -2293,24 +2291,7 @@ private boolean isAMLocked(int uid) {
 	}
 
 	private Context getContext() {
-		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)
```

Comment:
With the changes I made, the AM instance is now parsed to the service in all Android versions and all of them contains the "mContext" property, which is one of the few things that Google keeps consistent across the OS.